compiler/rustc_parse/src/parser/ty.rs

```diff
@@ -832,17 +841,31 @@ impl<'a> Parser<'a> {
 
     /// Emits an error if any trait bound modifiers were present.
     fn error_lt_bound_with_modifiers(&self, modifiers: BoundModifiers) {
-        if let Some(span) = modifiers.maybe_const {
-            self.sess.emit_err(errors::TildeConstLifetime { span });
+        match modifiers.constness {
+            BoundConstness::Never => {}
+            BoundConstness::Always(span) => {
+                self.sess.emit_err(errors::ModifierLifetime {
+                    span,
+                    modifier: "const",
+                    padding: " ",
+                });
```

Comment:
I did not see an ui test for the padding suggestion. Could you give me a link to it or add it?

Reply:
Well, `padding` didn't actually make sense. I've removed it. I could `run-rustfix` for the `-Zparse-only` test `tests/ui/parser/bounds-type.rs` but I'm not sure if it's worth it esp. since it concerns a maybe-incorrect suggestion.

Reply:
No need for run-rustfix, just an ui test that would have covered this code path with stderr would be fine. Though removing it also works.

Reply:
It's a tool-only suggestion, hence it doesn't show up on stderr.